test/api/v3/integration/user/auth/POST-login-local.test.js

```diff
@@ -43,7 +45,7 @@ describe('POST /user/auth/local/login', () => {
     })).to.eventually.be.rejected.and.eql({
       code: 401,
       error: 'NotAuthorized',
-      message: t('accountSuspended', { userId: user._id }),
+      message: t('accountSuspended', { communityManagerEmail: nconf.get('EMAILS:COMMUNITY_MANAGER_EMAIL'), userId: user._id }),
```

Comment:
I can certainly do that, though as it stands it's only called once in this whole test, or am I missing something?

If you want me to do this regardless, does adding the following in line 18 suffice (and modifying this line to use that variable instead)?
```javascript
let communityManagerEmail = nconf.get('EMAILS:COMMUNITY_MANAGER_EMAIL');
```

Lastly, I was only planning on doing this for the two test files:
* POST-login-local.test.js
* PUT-user_update_email.test.js

Or did you also want this in the two auth.js files? Doesn't seem like would add much value doing it there.